crates/subspace-node/src/bin/subspace-node.rs

```diff
@@ -369,6 +369,10 @@ fn main() -> Result<(), Error> {
                     .cloned();
 
                 let executor_enabled = !cli.secondary_chain_args.is_empty();
+                let block_import_throttling_buffer_size = match primary_chain_config.keep_blocks {
+                    KeepBlocks::All => 128, // TODO: proper value
+                    KeepBlocks::Some(n) => n,
+                };
```

Comment:
We initialize this to 1024 in the official documentation, certainly too much, I think the value of this buffer should be something small like 10, just enough for async block imports of both chains to work in parallel.